website/content/api-docs/index.mdx

```diff
@@ -75,6 +75,29 @@ $ curl \
     http://127.0.0.1:8500/v1/kv/foo
 ```
 
+## Accepts URL-Encoded Resource Names
```

Comment:
Does this section seem to be in the right place on this page? Or should it be higher/lower?